examples/slice-pizza/scripts/bootstrap.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env sh
+
+bootstrap() {
```

Comment:
The goal here is to simplify the `.env` and `config.js` setup for anyone who wants to clone and run this example.

The script asks for the various keys (which are read in safely without echoing to `stdout`) and then populates the files with the appropriate values.

It also provides direct links to the relevant URLs where you can find the various keys.